sql/moz-fx-data-shared-prod/search_terms_derived/merino_log_sanitized_v3/metadata.yaml

```diff
@@ -0,0 +1,17 @@
+friendly_name: Sanitized Firefox Search Terms Straight from Search Requests
+owners:
+  - ctroy@mozilla.com
+labels:
+  incremental: true
+description: |-
+  A sanitized variant of search terms submitted to Firefox.
+
+  This table allows removal of PII and slightly longer retention than
+  the underlying ping table.
+workgroup_access:
```

Comment:
This is missing BQ time partitioning present in `init.sql` e.g.

```
bigquery:
  time_partitioning:
    field: submission_timestamp
    type: day
    require_partition_filter: true
    expiration_days: 15
```

Reply:
this still appears to be missing from `merino_log_sanitized_v3`, though I see it was added to `suggest_impression_sanitized_v3` (where it was also missing but was meant to be added from https://github.com/mozilla/bigquery-etl/pull/3012#discussion_r890457882)

Reply:
```suggestion
bigquery:
  time_partitioning:
    field: timestamp
    type: day
    require_partition_filter: true
    expiration_days: 15
workgroup_access:
```